spec.html

```diff
@@ -35052,6 +35052,18 @@
               1. Let _onRejected_ be *undefined*.
             1. Let _fulfillReaction_ be the PromiseReaction { [[Capabilities]]: _resultCapability_, [[Type]]: `"Fulfill"`, [[Handler]]: _onFulfilled_ }.
             1. Let _rejectReaction_ be the PromiseReaction { [[Capabilities]]: _resultCapability_, [[Type]]: `"Reject"`, [[Handler]]: _onRejected_ }.
+            1. Return EnqueuePromiseReactions(_promise_, _fulfillReaction_, _rejectReaction_, _resultCapability_).
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-enqueuepromisereactions" aoid="EnqueuePromiseReactions">
+          <h1>EnqueuePromiseReactions ( _promise_, _fulfillReaction_, _rejectReaction_, _resultCapability_ )</h1>
+          <p>The abstract operation EnqueuePromiseReactions enqueues PromiseJobs with the provided PromiseReaction records on _promise_. The result is _resultCapability_'s promise.</p>
```

Comment:
"It returns" instead of "the result is", perhaps? I've never heard of an abstract operation's result before (although I bet if I Ctrl+F I'll find some... I'm just saying it seems rare enough I don't remember it).